### PR TITLE
DOCSP-27358 pt3 Leafy Green updates

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
## DESCRIPTION
Leafy green UI updates

## STAGING
Manage Saved Queries: https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-27358/query/queries/

Analyze Your Data Schema: https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-27358/schema/

Export Your Schema: https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-27358/schema/export/

In-Use Encryption Tutorial: https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-27358/in-use-encryption-tutorial/

Export to Specifiic Language: https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-27358/export-pipeline-to-language/

## JIRA
https://jira.mongodb.org/browse/DOCSP-27358

## BUILD LOG
https://jira.mongodb.org/browse/DOCSP-27358

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Comment by jocelyn-mendez1** [comment]
Hi @sarah-olson-mongodb, here is the final Leafy Green update. Let me know of any possible changes. Thanks in advance!

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks for these updates @jocelyn-mendez1! 
LGTM

File: source/includes/fact-favorite-query.rst [hunk 1]
```diff
@@ -1,29 +1,15 @@
 .. procedure::
    :style: normal
 
-   .. step:: Open query history
+   .. step:: Open query history.
 
       Click the :guilabel:`History` button at the top of the pipeline 
       builder.
```
Comment (Anemy) [inline source/includes/fact-favorite-query.rst:7]: Not from these changes, but I'm thinking we can fix this up here: 
```suggestion
      Click the :guilabel:`History` button on the query bar.
```
The `query/recent/#view-recent-queries` page or the screenshot below in `query/queries/#using-the-favorites-tab` might be nice to reference here as well.

File: source/includes/fact-save-pipeline.rst [hunk 2]
```diff
@@ -8,17 +8,10 @@ To save your pipeline:
...
+   .. step:: Click the :guilabel:`Save` dropdown button.
 
-      Click the :guilabel:`Save` dropdown button at the top of the pipeline
-      builder. Click on ``Save pipeline as...``
-
-      .. figure:: /images/compass/query-save-pipeline-as.png
-         :alt: Save pipeline as
+      Click on ``Save pipeline as...``.
 
    .. step:: Enter a name for your pipeline
```
Comment (Anemy) [inline source/includes/fact-save-pipeline.rst:15]: For consistency
```suggestion
   .. step:: Enter a name for your pipeline.
```

File: source/includes/fact-save-pipeline.rst [hunk 3]
```diff
@@ -8,17 +8,10 @@ To save your pipeline:
...
 
-      .. figure:: /images/compass/query-save-pipeline-as-name.png
-         :alt: Name your pipeline
-
    .. step:: Click :guilabel:`Save`
```
Comment (Anemy) [inline source/includes/fact-save-pipeline.rst:17]: Here as well
```suggestion
   .. step:: Click :guilabel:`Save`.
```

File: source/includes/steps-export-pipeline-to-language.yaml [hunk 4]
```diff
@@ -15,13 +15,9 @@ content: |
   |compass|, see :ref:`Aggregation Pipeline Builder
   <compass-agg-builder>`.
 ---
-title: Click the :guilabel:`Export` button.
+title: Click :guilabel:`Export to Language`.
```
Comment (Anemy) [inline source/includes/steps-export-pipeline-to-language.yaml:18]: Nice fix 👍

**Review by Anemy — APPROVED** [review]
Left a couple small suggestions, no blockers, lgtm!

**Review by ankit-gautam23 — APPROVED** [review]
LGTM

File: source/includes/fact-favorite-query.rst (hunk 1 above)
Comment (jocelyn-mendez1) [reply source/includes/fact-favorite-query.rst]: Done.

File: source/includes/fact-save-pipeline.rst (hunk 2 above)
Comment (jocelyn-mendez1) [reply source/includes/fact-save-pipeline.rst]: Done.

File: source/includes/fact-save-pipeline.rst (hunk 3 above)
Comment (jocelyn-mendez1) [reply source/includes/fact-save-pipeline.rst]: Done.

**Pull request merged**